scripts/build.ps1

```diff
@@ -24,6 +24,9 @@ This script provides helpers for building msquic.
 .PARAMETER DisableLogs
     Disables log collection.
 
+.PARAMETER StdoutLogs
+    Log to stdout (Only supported on unix)
+
```

Comment:
We should make this match CMake options. Let's update this to set the logging type. It should be optional and default to an empty string. Only if it's set to a non-empty string do we pass it down to CMake.